src/modules/localStorage.js

```diff
@@ -0,0 +1,14 @@
+// Function to save tasks to local storage
+function saveTasksToLocalStorage(tasks) {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
```

Comment:
- [ ] Hello, I observed that the todos were only displayed in the UI; kindly ensure that the array of todos is stored in the local storage as well. Below is a screenshot 👇 

<img width="667" alt="Okota flyer-modified (6)" src="https://github.com/Umuhan/to-do-list/assets/103386183/c287313c-89dd-4d86-8e7a-99fa4466e94a">

Reply:
done